### PR TITLE
Action maps and actions/bindings column views

### PR DESCRIPTION
**Pull request by andrew-oc**
### Description

UIToolkit versions of the action maps and actions/bindings columns in the input actions editor as they were in the IMGUI input actions editor.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Actions/Composites/Vector3Composite.cs [hunk 1]
```diff
@@ -173,6 +175,22 @@ public override void OnGUI()
...
+        public override void OnDrawVisualElements(VisualElement root, Action onChangedCallback)
+        {
+            var modeField = new EnumField("Mode", target.mode)
+            {
+                tooltip = m_ModeLabel.text
+            };
+
+            modeField.RegisterValueChangedCallback(evt =>
+            {
+                target.mode = (Vector2Composite.Mode)evt.newValue;
```
Comment (jamesmcgill) [inline Packages/com.unity.inputsystem/InputSystem/Actions/Composites/Vector3Composite.cs:188]: Should it be Vector3Composite for this one?

**Review by jamesmcgill — APPROVED** [review]
look good to me

**Comment by jfreire-unity** [comment]
@andrew-oc is this PR still relevant? Can we merge or close?

**Comment by jamesmcgill** [comment]
Closing as contents of PR was staged into https://github.com/Unity-Technologies/InputSystem/pull/1661 and is now merged.

**Pull request closed without merging**